src/actions.ts

```diff
@@ -169,6 +171,16 @@ export class PlasmicAction {
       `${pm.cmd} plasmic sync --projects '${this.args.projectId}:${this.args.projectApiToken}' --yes`,
       this.opts
     );
+    const localizedStrings = this.args.localizedStrings?.match(/[\w\d\/._-]+\.([\w]+)$/);
```

Comment:
Please add the caret anchor (`^`) to the regex to ensure it will match the file path completely: `/^[\w\d\/._-]+\.([\w]+)$/`

Reply:
added here: https://github.com/plasmicapp/plasmic-action/pull/3/files#diff-1b65654b4e6a4ad772c76520a607abec72eb17b56caaace589a3c5c51a21fdcbR176